hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/opt/physical/JoinNestedLoopPhysicalRel.java

```diff
@@ -109,6 +114,45 @@ public <V> V accept(CreateDagVisitor<V> visitor) {
         return visitor.onNestedLoopJoin(this);
     }
 
+    /**
+     * Cost calculation of Nested Loop Join relation.
+     * <p>
+     * Nested Loop Join algorithm is a simple join algorithm, where for each left row we are
+     * traversing the whole right row set. Cost estimation is the following: <ol>
+     * <li> Produced row count is L + R assuming the join selectivity is 1.
+     * <li> Processed row count is L * R because for each left row we are probing full right row set.
+     * <li> CPU is L * R * (row comparison cost). </ol>
+     * <p>
+     * A perfect estimation must also include memory and IO costs, as well as a selectivity
+     * for the right row set.
```

Comment:
```suggestion
     * Nested Loop Join algorithm is a simple join algorithm, where for each left row, we are
     * traversing the whole right row set. Cost estimation is the following: <ol>
     * <li> Produced row count is L * R * <i>join selectivity</i>.
     * <li> Processed row count is L * <i>k</i> * R, where <i>k</i> is 1 for non-equi-join,
     *      <i>join selectivity</i> &le; <i>k</i> &le; 1 for equi-join and 1/R for key lookup.
     * <li> CPU is L * <i>join selectivity</i> * R * <i>row comparison cost</i> assuming
     *      <i>k</i> converges to the join selectivity on average. </ol>
     * <p>
     * A perfect estimation must also include memory and IO costs.
```

Reply:
Since it's not a public API, I prefer having docs suitable for reading in raw format.